tailormap-components/projects/core/src/lib/feature-form/form-copy/form-copy.component.ts

```diff
@@ -162,12 +174,12 @@ export class FormCopyComponent implements OnInit, OnDestroy {
   }
 
   public isFieldChecked(event: any) {
-    const fieldsToCopy = this.formCopyService.featuresToCopy.get(this.originalFeature['objectGuid']);
+    const fieldsToCopy = this.formCopyService.featuresToCopy.get(this.originalFeature.objectGuid);
     return fieldsToCopy.has(event);
   }
 
   public isEverythingChecked(tab: string): boolean {
-    const fieldsToCopy = this.formCopyService.featuresToCopy.get(this.originalFeature['objectGuid']);
```

Comment:
not sure if this will pass the test of time: it is now very tightly coupled with GBI features (they contain an objectGuid). For future purposes it might be better to create a field (lets call it __fid) where we store the primary key and use that. For now, I think it will do